contracts/JBETHERC20SplitsPayer.sol

```diff
@@ -150,7 +150,8 @@ contract JBETHERC20SplitsPayer is IJBSplitsPayer, JBETHERC20ProjectPayer, Reentr
       defaultSplitsGroup,
       JBTokens.ETH,
       address(this).balance,
-      18 // decimals.
+      18, // decimals.
+      msg.sender
```

Comment:
Should we have a default value here too? (ie `defaultBeneficiary == address(0) ? msg.sender : defaultBeneficiary` )

Reply:
(and same in pay/addToBalanceOf)

Reply:
nice. using the already-available `defaultBeneficiary`